corehq/apps/reports/templates/reports/reportdata/excel_export_data.html

```diff
@@ -1,388 +0,0 @@
-{% extends report_base|default:"reports/async/bootstrap2/default.html" %}
```

Comment:
based on this wiki page, i think the whole thing is dead: https://confluence.dimagi.com/display/commcarepublic/De-Identify+Data